CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 2.16.0-rc.0 / 2020-01-30
+
+* [CHANGE] PromQL: Remove MaxConcurrent from the PromQL engine opts #6712
+* [FEATURE] PromQL: Support trailing commas in grouping opts #6480
```

Comment:
Is it worth mentioning for end users?

Reply:
It's an ENHANCEMENT and user visible

Reply:
It's a change in PromQL syntax.

In case someone is wondering why a query works on 2.16 but not with earlier Prometheus versions, this might be good to have in the changelog.

The same could be said about #6612 .